src/ansys/fluent/core/services/field_data.py

```diff
@@ -998,8 +1160,11 @@ def get_surface_data(
 
         Returns
         -------
-        Dict[int, np.array]
-            Dictionary containing a map of surface IDs to surface data.
+        Union[Vertices, FacesConnectivity, FacesNormal, FacesCentroid,
+        Dict[int, Union[Vertices, FacesConnectivity, FacesNormal, FacesCentroid]]]
+             If a surface name is provided as input, face vertices, connectivity, normal, or centroid data.
+             If surface IDs are provided as input, a dictionary containing a map of surface IDs to face
+             vertices, connectivity, normal, or centroid data.
```

Comment:
```suggestion
             If a surface name is provided as input, face vertices, connectivity data, and normal or centroid data are returned.
             If surface IDs are provided as input, a dictionary containing a map of surface IDs to face
             vertices, connectivity data, and normal, or centroid data is returned.
```
Based on earlier recommendations. Given the length of the options here, it seemed to note that these are two different return values.